src/components/Buttons/CreateSession.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
Make this a general button that takes the text to display and the `onClick` as inputs

Reply:
Made this into a general button and replaced wherever needed